clients/admin-ui/src/features/messaging-templates/messaging-templates.slice.ts

```diff
@@ -0,0 +1,33 @@
+import { baseApi } from "~/features/common/api.slice";
+
+export type MessagingTemplate = {
+  key: string;
+  label: string;
+  content: {
+    subject: string;
+    body: string;
+  };
+};
+
+// Messaging Templates API
+const messagingTemplatesApi = baseApi.injectEndpoints({
+  endpoints: (build) => ({
+    getMessagingTemplates: build.query<MessagingTemplate[], void>({
+      query: () => ({ url: `messaging/templates/` }),
+      providesTags: () => ["Messaging Templates"],
+    }),
+    updateMessagingTemplates: build.mutation<string, MessagingTemplate[]>({
```

Comment:
Would this endpoint actually return the `BulkPutMessagingTemplateResponse` response?

Reply:
Yes! Updated to use the correct type